CMakeLists.txt

```diff
@@ -71,5 +71,6 @@ target_link_libraries(impl_arrangement simplicial_arrangement::simplicial_arrang
           Eigen3::Eigen robust_implicit_networks implicit_functions::implicit_functions)
  target_compile_features(implicit_network_tests PRIVATE cxx_std_20)
 # target_compile_definitions(implicit_network_tests PRIVATE CATCH_CONFIG_ENABLE_BENCHMARKING)
+add_definitions(-DTEST_FILE="${CMAKE_CURRENT_LIST_DIR}/examples/tests")
```

Comment:
Note that this add a definition globally (all other code, including client code, will be compiled with this definition). It is better to add definition to specific cmake target.

```cmake
target_compile_definitions(
    implicit_network_tests    # Target name
    PRIVATE                   # Private means the definition is limited to this target
    -DTEST_FILE="${CMAKE_CURRENT_LIST_DIR}/examples/tests"
)
```

Reply:
I see. Thank you.